auth0_flutter/lib/src/web/auth0_flutter_plugin_real.dart

```diff
@@ -23,20 +23,54 @@ class Auth0FlutterPlugin extends Auth0FlutterWebPlatform {
   Auth0FlutterWebClientProxy? clientProxy;
   UrlSearchProvider urlSearchProvider = () => window.location.search;
 
+  /// The app state that was passed through [loginWithRedirect]
+  /// and retrieved in [initialize].
+  ///
+  /// This object is always a Dart object, never a JS object.
+  ///
+  /// When the login completes with the redirect, the page is reloaded.
+  /// Thus clearing this object is not needed,
+  /// as the actual state is managed across reloads,
+  /// using the transaction manager.
+  Object? _appState;
```

Comment:
This property & getter are new. We provide these now, so that:
a) users can retrieve the app state
b) we do not need to change the API for initialize/onLoad (the app state is web only anyway? Or so I think?)